Turkish Translation

realbaycan opened the pull request:
404 words translated to Turkish

realbaycan committed:
Turkish Translation

404 words translated to Turkish

realbaycan committed:
Update ch10-git-internals.asc

gokaygurcan commented on ch10-git-internals.asc:
```diff
@@ -1,19 +1,17 @@
-
-The content-addressable filesystem layer is amazingly cool, so we'll cover that first in this chapter; then, you'll learn about the transport mechanisms and the repository maintenance tasks that you may eventually have to deal with.
+Git'in başlangıcında (çoğunlukla 1.5 öncesi), kullanıcı arayüzü çok daha karmaşıktı çünkü cilalı bir VCS yerine bu dosya sistemini vurguladı.
+Son birkaç yılda kullanıcı arayüzü, orada bulunan herhangi bir sistem kadar temiz ve kullanımı kolay hale gelinceye kadar arıtıldı; Bununla birlikte, stereotip, öğrenmek zor ve karmaşık olan erken Git kullanıcı arayüzü hakkında durmaktadır.
```
bununla birlikte

gokaygurcan commented on ch10-git-internals.asc:
```diff
@@ -31,11 +29,11 @@ include::book/10-git-internals/sections/maintenance.asc[]
-This chapter has covered a number of plumbing commands -- commands that are lower level and simpler than the porcelain commands you've learned about in the rest of the book.
-Understanding how Git works at a lower level should make it easier to understand why it's doing what it's doing and also to write your own tools and helper scripts to make your specific workflow work for you.
+Bu noktada, Git'in arka planda ne yaptığını ve bir dereceye kadar nasıl uygulandığını oldukça iyi anlamış olmalısınız.
+Bu bölüm, bir dizi sıhhi tesisat komutunu kapsıyor - kitapların geri kalanında öğrendiğiniz porselen komutlardan daha basit ve daha basit olan komutlar.
```
Buradaki "plumbing" kelimesinin tam olarak "sıhhi tesisat" olduğudan emin değilim. Daha ziyade oldukça düşük seviyeli komutlar olduğunu belirtmek için kullanılmış bir terim gibi. Şimdilik böyle kalsın, ilerde gerekirse güncelleriz. 
Alternatiflere açığım ;)

gokaygurcan commented on ch10-git-internals.asc:
```diff
@@ -31,11 +29,11 @@ include::book/10-git-internals/sections/maintenance.asc[]
-Git as a content-addressable filesystem is a very powerful tool that you can easily use as more than just a VCS.
-We hope you can use your newfound knowledge of Git internals to implement your own cool application of this technology and feel more comfortable using Git in more advanced ways.
+İçerik adresli bir dosya sistemi olan Git, yalnızca bir VCS'den daha kolay bir şekilde kullanabileceğiniz çok güçlü bir araçtır.
+Umarız bu teknolojinin kendi harika uygulamanızı uygulamak için Git internals hakkındaki yeni bilginizi kullanabilir ve Git'i daha gelişmiş yollarla daha rahat uygulayabilirsiniz.
```
"Git internals" başlıkta "Git Kısımları" olarak kullanılmış, burada olduğu gibi bırakılmış. Birine karar vermek güzel olacaktır.

realbaycan committed:
Update ch10-git-internals.asc

Hatalarımı söyleyip bana yardımcı olduğunuz için teşekkür ederim. Dediğiniz gibi çevirilerdeki düzeltmeleri yaptım.
